package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "svelte-spa-router",
-  "version": "3.3.0",
-  "description": "Router for SPAs using Svelte 3",
+  "version": "4.0.0",
```

Comment:
Fine with me this time, but as a general principle please note that upgrading a `version` in a PR adding a feature is not recommended :)